config/browser_sync.yml

```diff
@@ -1,9 +1,12 @@
 # default is: localhost
-#host: 'some host'
+host: 'app'
+hosts: 'app'
 
```

Comment:
Do we need both `host` and `hosts`? It seems the implementation is confused; maybe remove `hosts`, if it isn't required